aws/table_aws_batch_job.go

```diff
@@ -0,0 +1,331 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/batch"
+
+	batchv1 "github.com/aws/aws-sdk-go/service/batch"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBatchJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_batch_job",
+		Description: "AWS Batch Job",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("job_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getBatchJob,
+			Tags:    map[string]string{"service": "batch", "action": "DescribeJobs"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBatchJobs,
+			Tags:    map[string]string{"service": "batch", "action": "ListJobs"},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "job_queue",
+					Require: plugin.Required,
+				},
+				{
+					Name:    "status",
+					Require: plugin.Optional,
+				},
+				{
+					Name:    "job_id",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		HydrateConfig: []plugin.HydrateConfig{
+			{
+				Func: getBatchJob,
+				Tags: map[string]string{"service": "batch", "action": "DescribeJobs"},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(batchv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the job.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("JobArn"),
+			},
+			{
+				Name:        "job_id",
+				Description: "The job ID.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "job_name",
+				Description: "The job name.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("JobName"),
+			},
+			{
+				Name:        "created_at",
```

Comment:
We may need a transform function(for converting the UNIX timestamp to our expected format) for this column value to be populated. We are getting the column value as number "[createdAt](https://docs.aws.amazon.com/batch/latest/APIReference/API_JobSummary.html#Batch-Type-JobSummary-createdAt)". 

Please check the same for other columns if we get the Unix timestamp value for a timestamp column type.